api/src/test/java/org/openmrs/module/pihcore/status/StatusDataFunctionsTest.java

```diff
@@ -41,7 +51,7 @@ public void testVelocityExpressions() {
         testExpression(dataValues, today, "$fn.formatDate($fn.now(), 'yyyy-MM-dd')");
         testExpression(dataValues, "0", "$fn.monthsSince($lastWeek)");
         testExpression(dataValues, "1", "$fn.monthsSince($lastMonth)");
-        testExpression(dataValues, "13", "$fn.monthsSince($lastYear)");
+        testExpression(dataValues, "12", "$fn.monthsSince($lastYear)");
```

Comment:
Since we are starting with an new instance (and not one that is 8 days behind) this becomes 12 months, not 13 months, which seems more correct?  It's possible that there was a reason for doing it this way though, and we are going to start running into issues on other dates?